### DISCUSSION TITLE
posts/killbot-aging-legend/

### DISCUSSION THREAD

**Discussion by giscus** [MoldyMacaroniX/gd-website#55] [Comments]
# posts/killbot-aging-legend/

Killbot is a level whose layout was first premiered back in 2017, with the first recording of its layout being uploaded by SrGuillester on September 12th of that year.

https://www.dashword.net/posts/killbot-aging-legend/

**Comment by Dolphin-02** [comment]
I still never could figure out who ari is. i searched everywhere but no results relating to GD someone explain pls
also cool article
